CHANGELOG-2.x.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+- Fixed a bug in which if `themeConfig.algolia` is not defined, the custom searchbar won't appear.
```

Comment:
```suggestion
- Fixed a bug in which if `themeConfig.algolia` is not defined, the custom searchbar won't appear.

If you've swizzled Algolia `SearchBar` component before, please update your source code otherwise CSS might break
```js
- <Fragment>
+ <div className="navbar__search" key="search-box">
```
```